subsys/net/lib/openthread/platform/logging.c

```diff
@@ -39,6 +39,28 @@ static inline int log_translate(otLogLevel aLogLevel)
 	return -1;
 }
 
+#if defined(CONFIG_LOG_MODE_MINIMAL)
+uint32_t log_count_args(const char *fmt)
```

Comment:
Maybe instead of relying on the logger configuration, we could have an internal, `static` utility function to count arguments, used in either case?

Reply:
Sure, sounds more future proof.